# Name of the domain bridge, used for node naming and logging
name: turtlesim_bridge # TODO: Problem in domain 1, multiple nodes named turtlesim_bridge1
from_domain: 1
to_domain: 2
topics:
  # Bridge "/goal_pose" topic from domain ID 1 to domain ID 2
  goal_pose:
    type: geometry_msgs/msg/PoseStamped

  # Bridge "/distanceToTarget" topic from domain ID 2 to domain ID 1
  distanceToTarget:
    type: communication_test_interfaces/msg/DistanceToTarget
    reversed: true

  # Bridge "/assignedRobot" topic from domain ID 1 to domain ID 2
  assignedRobot:
    type: std_msgs/msg/Int8

  # Bridge "/turtle_marker" topic from domain ID 2 to domain ID 1
  turtle_marker:
    type: visualization_msgs/msg/Marker
    reversed: true